lib/miq_smis_client.rb

```diff
@@ -682,14 +682,7 @@ def self.dotNodeInfo(node)
     end
 
     def self.dotHealth(node)
-      return 'springgreen3' # XXX
```

Comment:
I would just delete this line and leave the rest. It was probably added for testing and never removed - hence the XXX comment.